### ISSUE TITLE
[BUG] order of items on the map popup makes no sense

### ISSUE THREAD

**Issue by livcheerful** [livcheerful/anywhereadventures#316]
**Describe the bug**
It goes from X -> 2nd section
TIMESTAMP: 1:30

**To Reproduce**
1. Open a map pin
2. navigate through. it goes "Close location details"->Title->"Copy"->Separator

**Expected Behavior**
It should be: "Close location details" -> Title -> Section 1 (READ) -> OPEN -> Section 2 (Visit) -> Open in external map app -> copy -> I'm here
